src/components/title/show/title-show.js

```diff
@@ -250,86 +255,98 @@ class TitleShow extends Component {
             />)
           }
           bodyContent={(
-
-            <Accordion
-              label={<Headline size="large" tag="h3"><FormattedMessage id="ui-eholdings.title.titleInformation" /></Headline>}
-              open={sections.titleShowTitleInformation}
-              id="titleShowTitleInformation"
-              onToggle={this.handleSectionToggle}
-            >
-              <KeyValueColumns>
-                <div>
-                  <ContributorsList data={model.contributors} />
-
-                  {model.edition && (
-                    <KeyValue label={<FormattedMessage id="ui-eholdings.title.edition" />}>
-                      <div data-test-eholdings-title-show-edition>
-                        {model.edition}
-                      </div>
-                    </KeyValue>
-                  )}
-
-                  {model.publisherName && (
-                    <KeyValue label={<FormattedMessage id="ui-eholdings.title.publisherName" />}>
-                      <div data-test-eholdings-title-show-publisher-name>
-                        {model.publisherName}
+            <>
+              <Accordion
+                label={<Headline size="large" tag="h3"><FormattedMessage id="ui-eholdings.title.titleInformation" /></Headline>}
+                open={sections.titleShowTitleInformation}
+                id="titleShowTitleInformation"
+                onToggle={this.handleSectionToggle}
+              >
+                <KeyValueColumns>
+                  <div>
```

Comment:
This `<div>` could be removed. If we need `<div>` to split some blocks, better to use `<Row>` and `<Col>`